From 8e51448ede4852653568ace0f8587f911dfbf8f2 Mon Sep 17 00:00:00 2001
From: progier389 <progier@redhat.com>
Date: Tue, 8 Aug 2023 17:27:16 +0200
Subject: [PATCH 11/11] Issue 4551 - Part 2 - Fix build warning of previous PR
 (#5888)

Fix build paged search resuilt PR warning in header fix

Issue: 4551

Reviewed by: @mreynolds389 Thanks

(cherry picked from commit 89c2de5093c4522366d16decf81c06d4e5f8f96a)
---
 ldap/servers/slapd/proto-slap.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ldap/servers/slapd/proto-slap.h b/ldap/servers/slapd/proto-slap.h
index 47465c4c5..bb1f8aea2 100644
--- a/ldap/servers/slapd/proto-slap.h
+++ b/ldap/servers/slapd/proto-slap.h
@@ -1551,8 +1551,8 @@ int slapd_do_all_nss_ssl_init(int slapd_exemode, int importexport_encrypt, int s
 /*
  * pagedresults.c
  */
-void pageresult_lock_init();
-void pageresult_lock_cleanup();
+void pageresult_lock_init(void);
+void pageresult_lock_cleanup(void);
 pthread_mutex_t *pageresult_lock_get_addr(Connection *conn);
 int pagedresults_parse_control_value(Slapi_PBlock *pb, struct berval *psbvp, ber_int_t *pagesize, int *index, Slapi_Backend *be);
 void pagedresults_set_response_control(Slapi_PBlock *pb, int iscritical, ber_int_t estimate, int curr_search_count, int index);
-- 
2.41.0

